### PR TITLE
[Lab02-win] Improve skeleton TODOs and structure

### PR DESCRIPTION
**Pull request by teodutu**
Changes:
- `crc.c:`
    - Added separate files for writing and reading to and from a file,
      together with TODOs to remind students to use loops for these
      actions
    - Removed any mention of CRC from the comments on task *b*, so as to
      make it clearer that any files should be supported
    - Added the output files for the tests
- `ls.c`:
    - Fixed TODO numbers
    - Added a comment about `.` and `..` when the `-R` argument is given

### CODE REVIEW

File: labs/lab02/skel/win/2-crc/crc.c [hunk 1]
```diff
@@ -20,6 +20,11 @@
 #define BUFSIZE 512
 #define CHUNKSIZE 32
 
+static void WriteCrc(int crc, HANDLE hWrite)
+{
```
Comment (elena19m) [inline labs/lab02/skel/win/2-crc/crc.c:24]: Should me add some vars here?

ex (or something similar): 
```
BOOL bRet;
DWORD dwBytesWritten, dwBytesToWrite, dwBytesTotalWritten;
```

File: labs/lab02/skel/win/2-crc/crc.c [hunk 2]
```diff
@@ -32,20 +37,20 @@ static void GenerateCrc(CHAR *sourceFile, CHAR *destFile)
...
 		/* TODO 1 - Test for end of file */
 
-		/* calculate crc for buf */
+		/* Calculate crc for buf */
 		crc = update_crc(crc, (unsigned char *) buf, bytesRead);
 	}
 
-	/* TODO 1 - Write crc to destination file */
+	/* Write crc to destination file */
+	WriteCrc(crc, hWrite);
```
Comment (elena19m) [inline labs/lab02/skel/win/2-crc/crc.c:55]: You should remove bytesWritten  from here.

File: labs/lab02/skel/win/2-crc/crc.c [hunk 3]
```diff
@@ -60,27 +65,45 @@ static DWORD GetSize(HANDLE file)
 	return dwSize;
 }
 
+static DWORD ReadChunk(CHAR *chunk, HANDLE *hFile)
```
Comment (elena19m) [inline labs/lab02/skel/win/2-crc/crc.c]: Why `HANDLE *hFile`? I think ` static DWORD ReadChunk(CHAR *chunk, HANDLE hFile)` would be better.

File: labs/lab02/skel/win/3-ls/ls.c [hunk 4]
```diff
@@ -54,40 +53,40 @@ static void ListFiles(TCHAR *path, BOOL bRecursive, BOOL bAll)
 	sprintf_s(extendedPath, MAXPATH, "%s\\*", path);
 
 	/*
-	 * TODO 1 -  Find the first file in the directory
+	 * TODO 1 - Find the first file in the directory
 	 * Use extendePath as argument
 	 */
```
Comment (elena19m) [inline labs/lab02/skel/win/3-ls/ls.c:58]: Fix typo: extende**d**Path instead of extendePath.

**Review by elena19m — DISMISSED** [review]

File: labs/lab02/skel/win/2-crc/crc.c (hunk 1 above)
Comment (teodutu) [reply labs/lab02/skel/win/2-crc/crc.c:24]: I also added similar variables to the `ReadChunk` function.

**Review by elena19m — APPROVED** [review]

**Pull request merged**